src/main/java/com/mg105/Main.java

```diff
@@ -32,7 +36,14 @@ public static void main(String[] args) {
     @Override
     public void start(Stage primaryStage) {
         // Set up the initial entities
-        GameState state = new GameState();
+        // Set up should probably be moved to private method(s) or separate class?
```

Comment:
I would argue this is okay (if this is the only place where it happens) because it's in the `main` function.

(you can leave this here)